src/common/area-tooltip.component.ts

```diff
@@ -195,4 +197,29 @@ export class AreaTooltip implements OnChanges {
     this.anchorOpacity[index] = 0;
   }
 
+  /** This is a more flexible version of the original template "{{tooltipItem.series}}: {{tooltipItem.value.toLocaleString()}}" */
+  getToolTipText(tooltipItem: any): string {
+    let result: string = '';
+    if (tooltipItem.series) {
+      result += tooltipItem.series;
+    } else {
+      result += '???';
+    }
+    result += ': ';
+    if (tooltipItem.value) {
+      result += tooltipItem.value.toLocaleString();
+    }
+    if (tooltipItem.min || tooltipItem.max) {
```

Comment:
Done. I've also added the "≤" and "≥" stuff I'd already implemented with the circle tooltip.